compiler-core/src/format/tests.rs

```diff
@@ -5549,3 +5549,40 @@ fn pipeline_inside_tuple_is_not_nested_if_only_item() {
 "#
     );
 }
+
+// github.com/gleam-lang/gleam/issues/2608
+#[test]
+fn comments_are_not_moved_out_of_bounded_expressions() {
```

Comment:
Could you make a test per thing being tested please 🙏

Reply:
Sure!